src/course-updates/CourseUpdates.test.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { AppProvider } from '@edx/frontend-platform/react';
+import { initializeMockApp } from '@edx/frontend-platform';
+import MockAdapter from 'axios-mock-adapter';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+import CourseUpdates from './CourseUpdates';
+import messages from './messages';
+import initializeStore from '../store';
+import { getCourseUpdatesApiUrl, getCourseHandoutApiUrl } from './data/api';
+import { courseUpdatesMock, courseHandoutsMock } from './__mocks__';
+
+let axiosMock;
+let store;
+const mockPathname = '/foo-bar';
+const courseId = '123';
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useLocation: () => ({
+    pathname: mockPathname,
+  }),
+}));
+
+// Mock the tinymce lib
```

Comment:
I think this comment is pointing the obvious. I suggest removing it.

Reply:
Done